Lib/CollectionViewPagingLayout.swift

```diff
@@ -43,7 +43,13 @@ public class CollectionViewPagingLayout: UICollectionViewLayout {
     }
     
     private var numberOfItems: Int {
-        collectionView?.numberOfItems(inSection: 0) ?? 0
+        var count = 0;
+        if let sectionCount = collectionView?.numberOfSections {
+            for i in (0 ..< sectionCount) {
+                count += collectionView?.numberOfItems(inSection: i) ?? 0
+            }
+        }
+        return count
```

Comment:
we can also replace it with this, do you agree?
```suggestion
      (0..<(collectionView?.numberOfSections ?? 0))
            .compactMap { collectionView?.numberOfItems(inSection: $0) }
            .reduce(0, +)
```